src/test/java/uk/gov/hmcts/reform/ccd/document/am/feign/CaseDocumentClientTest.java

```diff
@@ -45,22 +45,27 @@
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
 
-@SpringBootTest(classes = {CaseDocumentClientApi.class})
+@SpringBootTest(classes = {CaseDocumentClient.class, CaseDocumentClientApi.class})
 @TestPropertySource(properties = "case_document_am.url=http://localhost:5170")
 @EnableAutoConfiguration
 @AutoConfigureWireMock(port = 5170)
-class CaseDocumentClientApiTest {
+public class CaseDocumentClientTest {
+
+    private static final Classification PUBLIC = Classification.PUBLIC;
 
     private static final UUID DOCUMENT_ID = UUID.randomUUID();
 
     private static final String URL = "/cases/documents";
 
-    private static final Classification PUBLIC = Classification.PUBLIC;
     private static final String CASE_TYPE_ID = "aCaseTypeId";
     private static final String JURISDICTION = "aJurisdictionId";
 
     private static final String SERVICE_AUTHORISATION_KEY = "ServiceAuthorization";
-    private static final String SERVICE_AUTHORISATION_VALUE = "a Bearer s2s token";
+
+    private static final String BEARER = "Bearer ";
```

Comment:
please remove BEARER and TOKEN comments. Use "AUTHORISATION_VALUE" . looks like your merge corrupted them.